### PR TITLE
first init

### PR DESCRIPTION
**Pull request by Jay2cof**
created first terraform init

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.terraform
+.terraform.lock.hcl
+terraform.tfstate.backup
+terraform.tfstate
+
+.DS_Store
```
Comment (davidebschke) [inline .gitignore:6]: please make at the end of one code document a empty line . That make for that not a problem but older programming language have problem without an empty line at the end. I write this comment here but that is for all other code documents

**Review by davidebschke — APPROVED** [review]
I don't see any Problems . Very good but read my comment its helpful for later

File: main.tf [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            "Action": "sts:assumeRole,
+            "Principal": {
+                "Service": "lambda.amazonaws.com"
+            }
+            "Effect": "Allow",
+            "Sid": ""
+        },
+    ]
+}
+EOF
```
Comment (fabianschmauder) [inline main.tf:30]: are you using an own account? In the sandbox you will not be able to create a iam role

File: main.tf [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+}
```
Comment (fabianschmauder) [inline main.tf:31]: empty line

File: python/lambda.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+def lambda_handler(event, context):
+    message = 'Hello {} !' .format(event['key1'])
+    return {
+        'message : message'
+    }
```
Comment (fabianschmauder) [inline python/lambda.py:5]: empty line

File: python/lambda.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+def lambda_handler(event, context):
```
Comment (fabianschmauder) [inline python/lambda.py:1]: you have lambda code but no lambda function in terraform?

**Review by fabianschmauder — COMMENTED** [review]
a few comments :)

File: .gitignore (hunk 1 above)
Comment (Jay2cof) [reply .gitignore:6]: done 👍

File: main.tf (hunk 2 above)
Comment (Jay2cof) [reply main.tf:30]: yes i plan to use it for the final work, but yes i cant go further here....so its being edited.

File: main.tf (hunk 3 above)
Comment (Jay2cof) [reply main.tf:31]: done

File: python/lambda.py (hunk 4 above)
Comment (Jay2cof) [reply python/lambda.py:5]: done

File: python/lambda.py (hunk 5 above)
Comment (Jay2cof) [reply python/lambda.py:1]: i wanted to make lambda a second branch.👍

**Pull request merged**